test/config/100-kafka-ephemeral-triple-2.4.0.yaml

```diff
@@ -13,20 +13,13 @@ spec:
       offsets.topic.replication.factor: 1
       transaction.state.log.replication.factor: 1
       transaction.state.log.min.isr: 1
-      log.message.format.version: "2.3"
+      log.message.format.version: "2.4"
     storage:
-      type: jbod
-      volumes:
-        - id: 0
-          type: persistent-claim
-          size: 100Gi
-          deleteClaim: false
+      type: ephemeral
   zookeeper:
-    replicas: 1
+    replicas: 3
```

Comment:
For the record: this is just needed, b/c of `ephemeral` storage usage.

To quote @scholzj, from https://github.com/strimzi/strimzi-kafka-operator/pull/2487

> The ephemeral storage is just an emptyDir and can get easily lost for example when pod restarts, when you do a rolling update etc. This kinda kills Kafka, because it looses all the information stored there. With 3 Zoo nodes, they can recover from each other. So the probability of thi happening is much smaller. Changing the node count here to 1 would probably make this much worse experience for many users using it.

Still the `ephemeral` seems "good enough" for the test env.